packages/modals/src/elements/Body.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { HTMLAttributes } from 'react';
+import { StyledBody } from '../styled';
+import { useModalsContext } from '../utils/useModalContext';
```

Comment:
Could we name this `useModalContext` (singular) to match the file name?

Reply:
Oops, I forgot to update this. It should be singular now, rather than plural. Thanks!